systemtest/src/main/java/io/strimzi/systemtest/resources/crd/KafkaResource.java

```diff
@@ -239,8 +239,8 @@ private static Kafka waitFor(Kafka kafka) {
 
         LOGGER.info("Waiting for Kafka {} in namespace {}", kafkaCrName, namespace);
 
-        StatefulSetUtils.waitForAllStatefulSetPodsReady(KafkaResources.zookeeperStatefulSetName(kafkaCrName), kafka.getSpec().getZookeeper().getReplicas());
-        StatefulSetUtils.waitForAllStatefulSetPodsReady(KafkaResources.kafkaStatefulSetName(kafkaCrName), kafka.getSpec().getKafka().getReplicas());
+        StatefulSetUtils.waitForStatefulSetReady(KafkaResources.zookeeperStatefulSetName(kafkaCrName));
+        StatefulSetUtils.waitForStatefulSetReady(KafkaResources.kafkaStatefulSetName(kafkaCrName));
```

Comment:
Why did you remove wait for pods? Is STs always `NotReady` when pods are not ready?